### PR TITLE
Add Airthings BLE documentation

### PR DESCRIPTION
**Pull request by vincegio**
## Proposed change

Add documentation for Airthings BLE integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [X] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/77284
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3618
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by vincegio** [comment]
I added `ha_release: 2022.9`, but am unsure if It is too late already. If it is then I'll change it :)

File: source/_integrations/airthings_ble.markdown [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  - '@vincegio'
+ha_domain: airthings_ble
+ha_bluetooth: true
+ha_platforms:
+  - sensor
+ha_config_flow: true
+ha_integration_type: integration
+---
+
+Integrates Airthings BLE sensors into Home Assistant.
```
Comment (klaasnicolaas) [inline source/_integrations/airthings_ble.markdown:19]: Maybe you could tell a little more about what airthings is and a link to their website?

File: source/_integrations/airthings_ble.markdown [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## Prerequisites
+
+In order to use this integration, it is required to have working Bluetooth set up on the device running Home Assistant.
+
+It is good if you know the MAC address of your Airthings Device. As of writing, the Airthings app does not show you the address of a device. 
+
+There are some apps and scripts to find the MAC address of BLE devices. A hands-on approach to find which one it is is to add the device to Home Assistant and:
```
Comment (klaasnicolaas) [inline source/_integrations/airthings_ble.markdown]:
```suggestion
There are some apps and scripts to find the MAC address of BLE devices. A hands-on approach to find which one it is is to add the device to Home Assistant and:

```

File: source/_integrations/airthings_ble.markdown [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+ha_integration_type: integration
+---
+
+Integrates Airthings BLE sensors into Home Assistant.
+
+## Prerequisites
+
+In order to use this integration, it is required to have working Bluetooth set up on the device running Home Assistant.
+
+It is good if you know the MAC address of your Airthings Device. As of writing, the Airthings app does not show you the address of a device.
```
Comment (klaasnicolaas) [inline source/_integrations/airthings_ble.markdown]: Why is it important to find out the MAC address?

**Review by klaasnicolaas — DISMISSED** [review]
I've left some comments with questions, if those questions are answered in the text it would be clearer for the users.

File: source/_integrations/airthings_ble.markdown (hunk 3 above)
Comment (ismarslomic) [reply source/_integrations/airthings_ble.markdown]: I guess knowing the MAC address during setup is only important if you have more then one Airthings device. The MAC address is part of the auto generated device name, to make it unique and possible to differentiate the devices. To avoid over complicating this concern in the docs, specially because most users would just sit on one single device (I guess), I would make this as own chapter in the docs and inform the user with more then one device to check the MAC address labeled on the device.

Comment (vincegio) [reply source/_integrations/airthings_ble.markdown]: Actually, appears that it is not. I found that the serial number is given to us so can just point to where to find this no. on the device :)

**Comment by vincegio** [comment]
Re-wrote it a bit. The integration went from `{device_name} {last 6 chars of mac}` to `{device_name} ({last 6 of s/n}`.

File: source/_integrations/airthings_ble.markdown (hunk 3 above)
Comment (ismarslomic) [reply source/_integrations/airthings_ble.markdown]: You are damn right, @vincegio ! Now I remember all the struggle I had finding the MAC address of mine Airthings device, when using the custom component [airthings_wave](https://github.com/custom-components/sensor.airthings_wave) which could not auto detect my device.

**Comment by ismarslomic** [comment]
> Re-wrote it a bit. The integration went from `{device_name} {last 6 chars of mac}` to `{device_name} ({last 6 of s/n}`.

Good idea! Serial number is labeled on the device, and is also visible from the official Airthings Wave mobile app (Device settings > Device info > Serial number)

File: source/_integrations/airthings_ble.markdown (hunk 3 above)
Comment (vincegio) [reply source/_integrations/airthings_ble.markdown]: Yeah, should be easier with just S/N hehe

**Comment by vincegio** [comment]
> > Re-wrote it a bit. The integration went from `{device_name} {last 6 chars of mac}` to `{device_name} ({last 6 of s/n}`.
> 
> Good idea! Serial number is labeled on the device, and is also visible from the official Airthings Wave mobile app (Device settings > Device info > Serial number)

Ah right, will add that too.

**Comment by ismarslomic** [comment]
@vincegio would it be useful for the user to know which Airthings sensors would be available in HA with this integration? This depends on the specific device model, of course, but if all sensors are available, then this might be stated in the docs?

**Comment by vincegio** [comment]
> @vincegio would it be useful for the user to know which Airthings sensors would be available in HA with this integration? This depends on the specific device model, of course, but if all sensors are available, then this might be stated in the docs?

What do you mean? 
There are some devices under `Supported models`, but are you thinking about something else?

**Comment by ismarslomic** [comment]
> > @vincegio would it be useful for the user to know which Airthings sensors would be available in HA with this integration? This depends on the specific device model, of course, but if all sensors are available, then this might be stated in the docs?
> 
> What do you mean? There are some devices under `Supported models`, but are you thinking about something else?

I'm thinking on all the sensor entities in HA, that will be added for your Airthings device, when using this BLE integration. 

Example for mine Wave+
image

**Comment by vincegio** [comment]
Oh yeah! Was thinking about models. 
Yeah maybe that would be interesting info.

**Comment by vincegio** [comment]
A bit unsure of how to structure sensor information, maybe this list is good enough.

File: source/_integrations/airthings_ble.markdown [hunk 4]
```diff
@@ -37,4 +36,16 @@ This integration uses the last 6 digits of the serial number.
 
 - Wave
 - Wave+
-- Wave mini
\ No newline at end of file
+- Wave mini
+
+## Sensors
+
+Sensors exposed, depending on data:
```
Comment (ismarslomic) [inline source/_integrations/airthings_ble.markdown]: Maybe rephrase this:
> Sensor entities added to HA, depending on the device model

Comment (vincegio) [reply source/_integrations/airthings_ble.markdown]: good one

File: source/_integrations/airthings_ble.markdown [hunk 5]
```diff
@@ -0,0 +1,51 @@
+---
+title: Airthings BLE
+description: Instructions on how to set up Airthings Devices over Bluetooth LE.
+ha_category:
+  - Environment
+  - Sensor
+ha_release: '2022.9'
```
Comment (frenck) [inline source/_integrations/airthings_ble.markdown]:
```suggestion
ha_release: '2022.11'
```

File: source/_integrations/airthings_ble.markdown [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+Integrates Airthings BLE sensors into Home Assistant.
+
+[Airthings](https://www.airthings.com/) provide different sensors for measuring the air quality. The focus specially on Radon sensors.
+
+Requires Airthings hardware and a compatible Bluetooth dongle.
+
+{% include integrations/config_flow.md %}
+
+The Airthings BLE integration will automatically discover devices once the [Bluetooth](/integrations/bluetooth) integration is enabled and functional. This will include the device name and it's serial number.
```
Comment (frenck) [inline source/_integrations/airthings_ble.markdown]:
```suggestion
The Airthings BLE integration will automatically discover devices once the [Bluetooth](/integrations/bluetooth) integration is enabled and functional. This will include the device name and its serial number.
```

**Review by frenck — APPROVED** [review]
Thanks, @vincegio 👍
